sdcm/cluster_k8s/__init__.py

```diff
@@ -2259,6 +2259,13 @@ def wait_for_nodes_up_and_normal(self, nodes=None, verification_node=None, itera
     def check_nodes_up_and_normal(self, nodes=None, verification_node=None):
         super().check_nodes_up_and_normal(nodes=nodes, verification_node=verification_node)
 
```

Comment:
Why not use
https://github.com/scylladb/scylla-cluster-tests/blob/fe67579bccad4ab26835c2c5fcb8d959e5eab589/sdcm/cluster.py#L4249

It's basically does the same...

Reply:
changed